src/hooks/result/useCategoryFilterList.ts

```diff
@@ -14,79 +16,35 @@ type ReturnType = {
   onCheckItem(name: string): void
 }
 
-export const cateGoryList = [
-  {
-    code: '전체',
-    name: '전체',
-    selected: true
-  },
-  {
-    code: '남성패션/잡화',
-    name: '남성패션/잡화',
-    selected: false
-  },
-  {
-    code: '여성패션/잡화',
-    name: '여성패션/잡화',
-    selected: false
-  },
-  {
-    code: '게임',
-    name: '게임',
-    selected: false
-  },
-  {
-    code: '스포츠/레저',
-    name: '스포츠/레저',
-    selected: false
-  },
-  {
-    code: '장난감/취미',
-    name: '장난감/취미',
-    selected: false
-  },
-  {
-    code: '디지털기기',
-    name: '디지털기기',
-    selected: false
-  },
-  {
-    code: '자동차/공구',
-    name: '자동차/공구',
-    selected: false
-  },
-  {
-    code: '생활가전',
-    name: '생활가전',
-    selected: false
-  },
-  {
-    code: '가구/인테리어',
-    name: '가구/인테리어',
-    selected: false
-  },
-  {
-    code: '도서/티켓/음반',
-    name: '도서/티켓/음반',
-    selected: false
-  },
-  {
-    code: '반려동물',
-    name: '반려동물',
-    selected: false
-  }
-]
-
 const useCategoryFilterList = (): ReturnType => {
-  const [list, setList] = useState<CheckItemType[]>(cateGoryList)
+  const { data: categories, isSuccess } = useGetCategoriesQuery()
```

Comment:
```suggestion
  const categories = useGetCategoriesQuery()
```
여기도 객체로 받는 형태가 더 가독성이 좋을 것 같습니다!

Reply:
[완료!](https://github.com/price-offer/offer-fe/pull/174/commits/2a5a1b48d520b38eefb1a97000a0f2136b488a10#diff-a3ef8cae97136a88c7de463df7fe5d4e73f2c431af0c1f779c5d28b846a5f7ef)